### ISSUE TITLE
Visual name and programmatically defined name for "Edit in settings.json" button is different.: A11y_Azure Machine Learning Extension for VS Code_Settings_label In Name

### ISSUE THREAD

**Issue by kupatkar99** [microsoft/vscode-tools-for-ai#2378]
"[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier."

### GitHub Tags
#A11yTCS; #A11ySev3; #WCAG2.5.3; #BM_AzureMachineLearningExtensionforVSCode_Win32_May2024; #DesktopApp; #Azure Machine Learning Extension for VS Code; #A11yMAS; #label In Name; #A11yWCAG2.1; #Win11; 

### Environment Details:
Application Name: Azure Machine Learning Extension for VS Code
Visual Studio Code: Version 1.89.1(User setup)

### Repro Steps:

1. Install VS Code
2. Install the Azure Machine Learning extension and sign in with appropriate Credentials.
3. Open the Command Palette via the keyboard command Ctrl+Shift+P (or Command+Shift+P for MacOS)
4. Type "Open User Settings" and select the option from the dropdown list called "Preferences: Open User Settings") and press enter
5. In the search box, type "Azure Machine Learning"
6. Tab until the list of settings is active
7. Use the up/down arrows to navigate between settings
8. Tab till "Edit in settings.json" button and Verify whether the Visual name and programmatically defined name for "Edit in settings.json" button is same or not.

### Actual Result:
Visual name and programmatically defined name for "Edit in settings.json" button is different.

### Expected Result:
Visual name and programmatically defined name for "Edit in settings.json" button should be same.

### User Impact:
Screen reader users will get confuse if extra information is being announced by screen reader.

### Attachment:
![A11y_Azure Machine Learning Extension for VS Code_Settings_label In Name](https://github.com/microsoft/vscode-tools-for-ai/assets/98530069/4f8084d0-2ebb-4086-b363-6cc23a3bc09e)

**Comment by kupatkar99** [comment]
Closing the bug as logged invalid

**Comment by kupatkar99** [comment]
#A11yTCSInvalid;
